From: Petr Tesarik <ptesarik@suse.com>
Date: Thu, 1 Apr 2021 15:09:52 +0200
Subject: Add 'bootdev=' to dracut command line
References: bsc#1182309
Git-commit: fe0777beff90a2479b451f5ef9e918249277c8ff

If this parameter is missing, dracut will parse the ip= command
line option(s) and create the corresponding initqueue jobs, but it
will not actually run them, unless they are needed to mount the
root filesystem.

Fixes: 2ad80dd96b91b889db44dd8c955cb127840a2be2
Signed-off-by: Petr Tesarik <ptesarik@suse.com>
---
 init/module-setup.sh |    1 +
 1 file changed, 1 insertion(+)

--- a/init/module-setup.sh
+++ b/init/module-setup.sh
@@ -213,6 +213,7 @@ kdump_cmdline_ip() {
 
     kdump_ifname_config "$kdump_host_if"
 
+    echo -n "bootdev=$kdump_iface"
     echo -n "$kdump_netif"
 
     case "$kdump_net_mode" in
